harmonica/isostasy.py

```diff
@@ -0,0 +1,54 @@
+"""
+Function to calculate the moho undulation assuming the Airy isostatic hypothesis.
+"""
+import numpy as np
+
+
+def isostasy_airy(topography, density_crust, density_mantle, density_water=None):
+    """
+    Computes the moho undulation using the Airy hypothesis.
+
+    On continental points:
+
+    .. math ::
+        m = \frac{\rho_c}{\rho_m - \rho_c} h
+
+    On oceanic points:
+
+    .. math ::
+        m = \frac{\rho_c - \rho_w}{\rho_m - \rho_c} h
+
+    where $L$ is the topography, $rho_m$ is the density of the mantle,
+    $rho_w$ is the density of the water, $\rho_c$ is the crustal density.
+
+    Parameters
+    ----------
+    density_mantle : float
+        Mantle density in kg/m$^3$.
```

Comment:
These `$` won't render into equations. Sphinx would require using `` :math:`kg/m^3` ``. Or you could use the ³ character, which is probably better in this case.